foundry.toml

```diff
@@ -26,17 +26,17 @@ bnb = "${RPC_BNB}"
 scroll="${RPC_SCROLL}"
 
 [etherscan]
-mainnet = { key="${ETHERSCAN_API_KEY_MAINNET}", chainId=1 }
```

Comment:
seems like chainId is wrong: https://book.getfoundry.sh/reference/config/etherscan not sure if was changed or has always been wrong